nemo/collections/tts/models/magpietts.py

```diff
@@ -252,8 +253,20 @@ def state_dict(self, destination=None, prefix='', keep_vars=False):
         return state_dict
 
     def load_state_dict(self, state_dict, strict=True):
+        if strict == False:
+            super().load_state_dict(state_dict, strict=False)
         # Override to load all the keys except _speaker_verification_model and _codec_model
-        super().load_state_dict(state_dict, strict=False)
+        for name, child in self.named_children():
+            if name in ['_speaker_verification_model', '_codec_model']:
+                continue
+            if any(param.numel() > 0 for param in child.parameters()):
+                new_state_dict = {}
+                for key in state_dict.keys():
+                    if key.startswith(name):
+                        new_state_dict[key[len(name)+1:]] = state_dict[key]
+                print(name)
+                print(new_state_dict)
```

Comment:
Looks good to me. Remove prints. Adding a comment could be helpful to explain eg. "decoder.char_embedding" -> "char_embedding"

Reply:
Done, can you resolve and re-review?